CHANGELOG.md

```diff
@@ -1,4 +1,40 @@
-## 4.71.0 (Unreleased)
+## 4.72.0 (Unreleased)
+
+FEATURES:
+* **New Resource:** `google_certificate_manager_public_ca_external_account_key` ([#14983](https://github.com/hashicorp/terraform-provider-google/pull/14983))
+
+IMPROVEMENTS:
+* compute: added `provisioned_throughput` field to `google_compute_disk` used by `hyperdisk-throughput` pd type ([#14985](https://github.com/hashicorp/terraform-provider-google/pull/14985))
+* container: added field `security_posture_config` to resource `google_container_cluster` ([#14999](https://github.com/hashicorp/terraform-provider-google/pull/14999))
```

Comment:
Do we want consistency between this and the above line? "added field x" vs "added x field"

Reply:
Ehh- both follow the style guide as-written.